packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -40,6 +42,18 @@ type Conclusion =
   | 'action_required'
   | undefined;
 
+type ChangeTypes = 'add' | 'del';
+
+interface Change {
+  type: ChangeTypes;
+  region_tag: string;
```

Comment:
suggestion: `regionTag` to keep the casing consistent for the keys that we control.

Reply:
Done